task2/js/app.js

```diff
@@ -0,0 +1,33 @@
+document.addEventListener('DOMContentLoaded', () => {
+    let imageName = 'zombie-dead';
+    const imagePath = '/icons';
+    const extension = 'gif';
+    const zombieParentSelector = document.querySelector('.zombies');
+    const zombieSrc = `${imagePath}/${imageName}.${extension}`;
```

Comment:
It's better to use CSS class instead of using image path:
```
.zombie {}
.zombie-strong { // background image }
...

```

The reason for this is split JS code and CSS style to allow define style and script separately